src/libsync/bulkpropagatorjob.h

```diff
@@ -163,6 +165,8 @@ private slots:
 
     std::vector<BulkUploadItem> _filesToUpload;
 
+    std::atomic<qint64> _sentTotal = 0;
```

Comment:
are you sure this is needed ?
do we really access `_sentTotal` from multiple threads ?